src/generate-elements.ts

```diff
@@ -134,6 +147,29 @@ function serializeNamespace(
   return metadata;
 }
 
+function serializeFunction(
+    fn: ResolvedFunction, packagePath: string): Function {
+  const packageRelativePath =
+      pathLib.relative(packagePath, fn.sourceRange.file);
+  const metadata: Function = {
```

Comment:
Agreed, there should be a larger refactoring step to address this across the repo. Right now one is a class and the other is a model interface, and I actually went out of my way to not share code until we had a chance to address throughout.